src/__tests__/paxfulapi_refresh_test.ts

```diff
@@ -79,6 +81,61 @@ describe("With the Paxful API SDK", function () {
         })
     });
 
+    it("Client credentials are not saved if refresh response code is wrong", async () => {
+        fetchMock.once({
+            url: /oauth2\/userinfo/,
+            method: "GET"
+        }, {
```

Comment:
This test case one above, though expect to be testing different scenarios (wrong response code VS empty body), have mocks setup in the same way, e.g.:

`{
            status: 401,
            body: ""
        }`

In this specific test case we need to specify wrong code, but OK body. In the second on - empty body and OK code. Right?